src/common/index.ts

```diff
@@ -1,21 +1,10 @@
-import base64 from './base64'
-import blob from './blob'
-import constants from './constants'
-import typeChecks from './type-checks'
-import * as types from './types'
-import util from './util'
+import * as base64 from './base64'
+import * as blob from './blob'
 
-export { base64, blob }
-export * from './constants'
 export * from './types'
 export * from './type-checks'
 export * from './util'
+export { base64, blob }
 
-export default {
-  base64,
-  blob,
-  ...constants,
-  ...types,
-  ...typeChecks,
-  ...util
-}
```

Comment:
I removed these in a bunch of places, not in `fs/*` or `ipfs/*` (I think). Because I feel like we're doing additional work here, we should probably do one or the other. Either use `export function ...` or `export default { ... }`, no?

Reply:
:man_shrugging: I personally like the flexibility of either being able to import an entire module or just a function/variable from it. But you're right, it is more work, and it's very easy for your `default` export to get out of sync with the module. I don't have a strong opinion on it. I'm good with leaving it off unless we run into a situation where it would help

Reply:
Yeah, I'd just use `import * as module from` to import an entire module. That way you don't need to maintain that list. `import { somethin }, * as module from` works too.

Reply:
makes sense to me. I'm happy switching over to this convention :ok_hand: